charts/kangal/README.md

```diff
@@ -1,30 +1,38 @@
-# Kangal chart
-
+# Kangal Chart
 [Kangal](https://github.com/hellofresh/kangal) is a tool to spin up an isolated environment in a Kubernetes cluster to run performance tests using JMeter
 
-## TL;DR;
-```console
-$ helm repo add kangal https://hellofresh.github.io/kangal
-$ helm install kangal/kangal --name kangal
-$ kubectl apply -f  https://raw.githubusercontent.com/hellofresh/kangal/master/charts/kangal/crd.yaml
-```
-
 ## Introduction
-
 This chart bootstraps a [kangal](https://github.com/hellofresh/kangal) deployment using the [Helm](https://helm.sh) package manager.
 
 ## Prerequisites
-
-- Helm 2.13
+- Helm 3+
```

Comment:
we don't support helm3 in our chart

Reply:
It's working with helm3

Reply:
yes, but it is not required. You may put Helm 2+ maybe?

Reply:
Helm2 commands are different :(
I will change a bit and add a helm2 example

Reply:
Done!